Collector/MigrationsFlattener.php

```diff
@@ -11,30 +11,30 @@
 
 class MigrationsFlattener
 {
-    public function flattenAvailableMigrations(AvailableMigrationsList $migrationsList, ?ExecutedMigrationsList $executedMigrations = null): array
+    public function flattenNewMigrations(AvailableMigrationsList $migrationsList, ExecutedMigrationsList $executedMigrations): array
```

Comment:
overall looks very good, the only thing that confused me is the name of this method.

`flattenNewMigrations` gives the impression to flatten a list on new migrations, but the method is first calculating the list of new migrations using the executed and available migrations, and then is flattening them.

would it make sense to give a better name to the method or to extract the process of calculating the new migrations in a private method as part of the data collector class?

Reply:
yup, that's a good idea. wasn't sure about this anyways because it duplicates logic of `Doctrine\Migrations\Version\CurrentMigrationStatusCalculator::getNewMigrations`, but that would bring a refactoring to the migrations lib which felt like a scope creep.
what do you think about that?

Reply:
yea, i understand, but personally in think that is fine.